Added configuration for the new WIFI device sold by Obi

arthurf1969 opened the pull request:
There is a new version of the WIFI Schuko device sold by OBI (here in germany). This device uses an ESP-WROOM02 with two LED's (attached to GPIO12 and GPIO13) as well as a button attached to GPIO5.
I have also added an image and updated the readme file for this device.

mcspr commented on code/espurna/config/hardware.h:
```diff
@@ -3335,6 +3335,35 @@
+// This configuration is for the second generation of devices sold by OBI.
+// https://www.obi.de/hausfunksteuerung/wifi-stecker-schuko-weiss/p/4077806
+// -----------------------------------------------------------------------------
+#elif defined(OBI_OBI_V2)
```
Any reason why not name it the same as previous one?
`OBI_WIFI_STECKER_SCHUKO_V2`?

arthurf1969 replied:
No, I was just not aware of the brand behind the device. You're absolutely right this should be renamed (and probalby be relocated so that both are close together? What do you think?)

arthurf1969 commented:
I have changed the name of the configuration to EUROMATE_WIFI_STECKER_SCHUKO_V2, relocated the configuration next to the older device EUROMATE_WIFI_STECKER_SCHUKO and fixed the spelling of the word "Schuko" also in the configuration of the older device.

mcspr commented:
Thank you!

@xoseperez 
For the v1, #622 is the only mention of the EUROMATE I could find. Maybe just have OBI for both? https://github.com/arendst/Sonoff-Tasmota/pull/3944 links to the same store page and actually tracked down the manufacturer (which is not the same for this one): http://www.cn-yidong.com/Home/En-products-cid-18.html

arthurf1969 commented:
Just as an information for you: I originally did not take a closer look at the housing of the device. On the back of the device there is the name EUROMATE (injection molded in the housing - white on white ... silly is). Euromate is part of the OBI brand. Thus my original thought on OBI. But to me (similar to the version 1) this is just a rebranded device from some other chinese brand. In contrast to the first version it does not use a latching relay.

mcspr commented:
Well, then it makes sense to keep it like it is. 
Thank you for clarifying, as I was not aware of obi / euromate relation (slightly different google search would've revealed it:)
